### PR TITLE
docs(input validation): fix examples for MaskedTextbox and Slider

### PR DESCRIPTION
**Pull request by YanushevGeorgi**
Slider-related margin fix is due to https://github.com/telerik/kendo-themes/issues/4698

### CODE REVIEW

File: common-features/input-validation.md [hunk 1]
```diff
@@ -735,6 +741,12 @@ The sliders are, effectively, numeric inputs in terms of behavior and what data
     <TelerikButton ButtonType="@ButtonType.Submit">Submit</TelerikButton>
 </EditForm>
 
+<style>
+    .k-slider{
+        height: 60px
```
Comment (dimodi) [inline common-features/input-validation.md:746]: If the purpose of this CSS rule is to increase the space between the Sliders and the labels, there are better ways to do that, which do not include our internal CSS classes - for example, add margins to the custom labels.

File: common-features/input-validation.md [hunk 2]
```diff
@@ -531,29 +531,35 @@ You may want to set the [`IncludeLiterals`]({%slug maskedtextbox-mask-prompt%}#i
     <ValidationSummary />
 
     <p>
-        <TelerikMaskedTextBox Mask="0000-0000-0000-0000"
-                              IncludeLiterals="true"
-                              Label="Credit Card:"
-                              @bind-Value="@Payment.CreditCard">
-        </TelerikMaskedTextBox>
+        <TelerikFloatingLabel Text="Credit Card*:">
```
Comment (dimodi) [inline common-features/input-validation.md:534]: We don't use `*` for required fields anywhere.

```suggestion
        <TelerikFloatingLabel Text="Credit Card:">
```

**Review by dimodi — CHANGES_REQUESTED** [review]
Replace the Slider height with some other CSS, which doesn't require knowledge about our components' CSS classes.

**Review by yordan-mitev — COMMENTED** [review]
✨

File: common-features/input-validation.md [hunk 3]
```diff
@@ -735,6 +741,12 @@ The sliders are, effectively, numeric inputs in terms of behavior and what data
     <TelerikButton ButtonType="@ButtonType.Submit">Submit</TelerikButton>
 </EditForm>
 
+<style>
+    .validation-message {
+        margin-top: 15px
```
Comment (dimodi) [inline common-features/input-validation.md:746]: Always use `;`

```suggestion
        margin-top: 1em;
```

**Review by dimodi — APPROVED** [review]

**Pull request merged**